.template/spec/base/template_spec.rb

```diff
@@ -26,4 +20,8 @@
     expect(file('Procfile')).to exist
     expect(file('Procfile.dev')).to exist
   end
+
+  it 'creates Reek configuration files' do
+    expect(file('.reek.yml')).to exist
```

Comment:
should we add the test for the `Dangerfile` too 🤔

Reply:
I did add test for `Dangerfile` but I added it mistakenly for the web variant only. Fixing it right away.